mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -110,6 +110,18 @@ properties:
       - :DSTU2
       - :STU3
       - :R4
+  - !ruby/object:Api::Type::Enum
+    name: complexDataTypeReferenceParsing
+    description: |
+      Enable parsing of references within complex FHIR data types such as Extensions. If this value is set to ENABLED, then features like referential integrity and Bundle reference rewriting apply to all references. If this flag has not been specified the behavior of the FHIR store will not change, references in complex data types will not be parsed. New stores will have this value set to ENABLED after a notification period. Warning: turning on this flag causes processing existing resources to fail if they contain references to non-existent resources.
+    exact_version: ga
+    required: false
+    immutable: false
```

Comment:
nit: Can be omitted, does not differ from default.

Reply:
Done.